docs/install/README.md

```diff
@@ -1,57 +1,93 @@
-## Installing Knative with GCP
+# Installing Knative with GCP
 
-To install Knative with GCP from master,
+1. Create a
+   [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+   and install the `gcloud` CLI and run `gcloud auth login`. This guide will
+   use a mix of `gcloud` and `kubectl` commands. The rest of the guide assumes
+   that you've set the `PROJECT_ID` environment variable to your Google Cloud
+   project id, and also set your project ID as default using
+   `gcloud config set project $PROJECT_ID`.
 
-### Using [ko](http://github.com/google/ko)
+1. Install [Knative](https://knative.dev/docs/install/). Preferably, set up both [Serving](https://knative.dev/docs/serving/)
+   and [Eventing](https://knative.dev/docs/eventing/). The latter is only required if you want to use the Pub/Sub `Channel`.  
 
-```shell
-ko apply -f ./config
-```
+1. Create the `cloud-run-events` namespace. This is the namespace where our control plane pods run.
 
-Applying the `config` dir will:
+     ```shell
+     kubectl create namespace cloud-run-events
```

Comment:
Do we need to create the namespace in advance?

Reply:
yes, because the secret needs to be created in that namespace, and that needs to be done before we apply the yamls.. I'd still like to keep the namespace yaml there, because this might change in the near future, and it doesn't hurt